source/code-generation/query-converter/test-converted-queries/install-query-runner.txt

```diff
@@ -17,4 +17,100 @@ Install Query Runner
    :depth: 1
    :class: singlecol
 
-Task page.
\ No newline at end of file
+To use the :guilabel:`Test Queries` pane , Relational
+Migrator must have access to the Query Runner. 
+Query Runner is a Docker container with 
+all the dependencies needed to test your converted 
+queries.
+
+About this Task
+---------------
+
+- If the version of Query Runner is different than the 
+  version of Relational Migrator a warning banner displays.
+  
+- To update Query Runner to the latest version, run 
+  the ``docker pull`` command from step number one.
+
+- Query Runner operates using the ``http`` protocol, 
+  requests made from a remote machine are not encrypted.
+
+- Query Runner can run either locally or remotely.
+
+Before you Begin
+----------------
+
+Install `Docker <https://www.docker.com/get-started/>`__.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Get the Query Runner Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+   .. step:: Run the Image as a Container
+
+         - To run the container locally:
+         
+           .. code-block:: sh
+
+              docker run --name relational-migrator-query-runner -p 127.0.0.1:6080:6080 public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+         - To run the container remotely:
+                     
+           a. Add the server address where the  
+              service is running to the ``migrator.language-runner.server.address`` in the 
+              ``user.properties``.
+
+              For example:
+
+              .. code-block:: sh
+
+                 migrator.language-runner.server.address: http://myserver:6080 
+
+              .. tip::
+                  
+                 For details on where the ``user.properties`` file is located, 
+                 see :ref:`file-location`. 
+
+           #. After changing the ``user.properties`` file restart Relational Migrator.
+
+           #. Launch the container from your remote server or container hosting service.
+
+              To launch the container on your remote server expose port ``6080`` 
+              with the following docker command:
+
+              .. code-block:: sh
+
+                 docker run --name relational-migrator-query-runner -p 6080:6080 public.ecr.aws/v4d7k6c9/relational-migrator-query-runner
+
+   .. step:: Confirm the Query Runner container is running
+
+      In your web browser navigate to the host and port the Query Runner 
+      container is running on. For example: 
```

Comment:
(suggestion)
"... is using."
Totally optional. It might help to reduce the repetition of "run".